requirements_dev.in

```diff
@@ -21,3 +21,4 @@ rcssmin==1.1.1
 six==1.16.0
 typing-extensions==4.0.1
 webtest==3.0.0
+webptools==0.0.9
```

Comment:
I don't think it needs to be added here if we add it to _requirements.in_

Reply:
Updated